---
title: "Combined analysis of transposable elements and structural variation in maize genomes reveals genome contraction outpaces expansion"

authors:

  - Manisha Munasinghe, Ph.D.
  - Andrew Read, Ph.D.
  - Michelle C. Stitzer, Ph.D.
  - Baoxing Song, Ph.D.
  - Claire Menard
  - Kristy Yubo Ma
  - Yaniv Brandvain, Ph.D.
  - Candice N. Hirsch, Ph.D.
  - Nathan Springer, Ph.D.

date: "2019-04-07T00:00:00Z"
doi: ""

# Schedule page publish date (NOT publication's date).
publish Date: "2024-01-01T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "PLoS Genetics"
publication_short: "PLoS Genet"

abstract: Structural differences between genomes are a major source of genetic variation that contributes to phenotypic differences. Transposable elements, mobile genetic sequences capable of increasing their copy number and propagating themselves within genomes, can generate structural variation. However, their repetitive nature makes it difficult to characterize fine-scale differences in their presence at specific positions, limiting our understanding of their impact on genome variation. Domesticated maize is a particularly good system for exploring the impact of transposable element proliferation as over 70% of the genome is annotated as transposable elements. High-quality transposable element annotations were recently generated for de-novo genome assemblies of 26 diverse inbred maize lines. We generated base-pair resolved pairwise alignments between the B73 maize reference genome and the remaining 25 inbred maize line assemblies. From this data, we classified transposable elements as either shared or polymorphic in a given pairwise comparison. Our analysis uncovered substantial structural variation between lines, representing both putative insertion and deletion events. Putative insertions in SNP depleted regions, which represent recently diverged identity by state blocks, suggest some TE families may still be active. However, our analysis reveals that, genome-wide, deletions of transposable elements account for more structural variation than insertions. These deletions are often large structural variants containing multiple transposable elements. Combined, our results highlight how transposable elements contribute to structural variation and demonstrate that deletion events are a major contributor to genomic differences.

# Summary. An optional shortened abstract.
summary: Mutation followed by selection is the basis for evolution and drives diversification. These mutations can be small single nucleotide changes or larger insertions or deletions of DNA sequence, referred to as structural variants. Structural variation is often associated with the repetitive nature and functional activity of transposable elements. Here, we used whole genome alignments to identify structural variation between the maize reference genome and 25 other inbred maize lines. We intersected these genomic structural variants with publicly available transposable element annotations to determine what proportion of the structural variation is annotated as transposable element. The make-up of transposable elements at each structural variant allowed us to predict whether the variant represented a likely insertion or deletion event. We find that many variants are likely deletion events composed of multiple transposable element types, as well as sequence that did not originate from a transposable element. Our results demonstrate the role that transposable elements play in generating genomic diversity and provide a valuable resource for future studies of diversity across transposon-rich maize genomes.

tags:
- Source Themes
featured: false

# links:
# - name: ""
#   url: ""
url_pdf: https://www.biorxiv.org/content/10.1101/2023.03.02.530873v1.full.pdf
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: 'https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1011086'
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: 'Image credit: [**Unsplash**](https://unsplash.com/photos/jdD8gXaTZsc)'
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: example
---

{{% callout note %}}
Click the *Cite* button above to demo the feature to enable visitors to import publication metadata into their reference management software.
{{% /callout %}}

{{% callout note %}}
Create your slides in Markdown - click the *Slides* button to check out the example.
{{% /callout %}}

Supplementary notes can be added here, including [code, math, and images](https://wowchemy.com/docs/writing-markdown-latex/).
